docs/usage.md

```diff
@@ -17,7 +17,7 @@ Refer to the section **OT - Vulnerability Management** to understand how this so
     </tr>
     <tr>
         <td>
-            Before using the simulation mode, make the following configurations are in place:
+            Before using the simulation mode, make the following changes:
             <ol>
                 <li>To use actual data of listed CVEs, configure the <strong><em>NIST National Vulnerability Database</em></strong> connector.</li>

```

Comment:
Please change "To use actual data of listed CVEs" to "To get latest report of listed CVEs"